nemo/collections/nlp/data/datasets/sgd_dataset/evaluate.py

```diff
@@ -123,6 +123,7 @@ def get_metrics(dataset_ref, dataset_hyp, service_schemas, in_domain_services, j
     per_frame_metric = {}
 
     for dial_id, dial_hyp in dataset_hyp.items():
+        unseen = False
```

Comment:
Replace unseen=False with seen=True, it'll be easier to read

Reply:
That one specifies if we have seen any unseen. so confusing. Changed it.

Reply:
chnaged to found_unseen?

Reply:
I'd just use `seen` var instead of `unseen` because you're checking `if not in unseen` which basically means `seen`